src/main/webapp/app/entities/programming-exercise/utils/programming-exercise.utils.ts

```diff
@@ -0,0 +1,11 @@
+// See: https://github.com/ls1intum/Artemis/commit/c842a8995f9f837b010d1ddfa3ebe00df7652011
+// We changed the notification plugin to also send information about successful tests (previously only failed tests).
+// In some cases it needs to be checked explicitly wether a result is legacy or not.
+// The date used is the date of the merge: 2019-05-10T22:12:28Z.
+import { Result } from 'app/entities/result';
+
+const BAMBOO_RESULT_LEGACY_TIMESTAMP = 1557526348000;
```

Comment:
I had to introduce this to differentiate properly between legacy and new results (see comment above).

Reply:
As discussed with Stephan we need this for now, but would be able to deactivate the check in the next semester.